Sources/SpeziAccount/SpeziAccount.docc/Account Values/Adding new Account Values.md

```diff
@@ -97,6 +97,26 @@ Therefore, you typically do not need to provide a custom view implementation,
 or you might consider adding `CustomLocalizedStringResourceConvertible` protocol
 conformance to your `Value` type.
 
+However, if your `Value` is not `String`-based, you need to implement your own DataDisplayView. Below is a short code example that demonstrates how to implement a DataDisplayView for which the `Value` is `Int`-based for a HeightValue Key.
+```swift
+public struct DataDisplay: DataDisplayView {
+    public typealias Key = HeightKey
+    private let height: Int
+    public init(_ value: Int) {
+        self.height = value
+    }
+    public var body: some View {
+        HStack {
+            Text(HeightKey.name)
+            Spacer()
+            Text("\(height) cm")
+                .foregroundColor(.secondary)
+        }
+        .accessibilityElement(children: .combine)
```

Comment:
Please be consistent with the indentation of modifiers. Indent this modifier similar to how you have done it with the `foregroundColor` modifier a few lines above.